ec/src/msm/variable_base/mod.rs

```diff
@@ -142,13 +142,12 @@ impl VariableBase {
         assert!(scalars_stream.len() <= bases_stream.len());
 
         // remove offset
-        let mut bases = bases_stream.iter();
+        let bases_init = bases_stream.iter();
         let mut scalars = scalars_stream.iter();
 
         // align the streams
-        bases
-            .advance_by(bases_stream.len() - scalars_stream.len())
-            .expect("bases not long enough");
+        let mut bases = bases_init.skip(bases_stream.len() - scalars_stream.len());
+        // .expect("bases not long enough");
```

Comment:
```suggestion
```

Reply:
This line was sort of left for the future change back to `advance_by`. But I think maybe we can just remove it for now.

Reply:
I'd rather write explicitly the issue on rust-lang that is keeping track of the implementation of `advance_by`

Reply:
That indeed sounds better!